app/models/grda_warehouse/import_threshold.rb

```diff
@@ -0,0 +1,189 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'memery'
+
+# This defines two thresholds for imports that will potentially trigger notifications
+# 1. When an import is going to add or remove a significant percentage of the existing records, and that change is greater than the min count threshold
+# 2. When an import contains a high percentage of errors, and that change is greater than the min count threshold
+module GrdaWarehouse
+  class ImportThreshold < GrdaWarehouseBase
+    include Memery
+    belongs_to :data_source
+
```

Comment:
maybe could use validation?

```suggestion

validates :record_count_change_min_threshold, numericality: { greater_than_or_equal_to: 0 }
validates :record_count_change_percent_threshold, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 100 }

```